crates/bevy_render/src/camera/camera.rs

```diff
@@ -74,12 +74,13 @@ impl RenderTarget {
             RenderTarget::Window(window_id) => windows
                 .get(*window_id)
                 .map(|window| UVec2::new(window.physical_width(), window.physical_height())),
-            RenderTarget::Image(image_handle) => images.get(image_handle).map(|image| {
-                let Extent3d { width, height, .. } = image.texture_descriptor.size;
-                UVec2::new(width, height)
-            }),
-        }
-        .filter(|size| size.x > 0 && size.y > 0)
+            RenderTarget::Image(image_handle) => images
+                .get(image_handle)
+                .map(|image| {
+                    let Extent3d { width, height, .. } = image.texture_descriptor.size;
+                    UVec2::new(width, height)
+                }),
+        }.filter(|dim| dim.x > 0 && dim.y > 0)
```

Comment:
I'm not sure I see what changed hhere apart from a variable name?

Reply:
This is odd. I was working from a version a few commits old, and had to merge those changes in, not sure where this came from. I'll try and clean up this PR's diff.